WeakAuras/Prototypes.lua

```diff
@@ -4118,6 +4118,8 @@ Private.event_prototypes = {
         local genericShowOn = %s
         local expirationTime = startTime and duration and startTime + duration
         state.spellname = spellname;
+        state.modRate = modRate;
+        state.modRateCharges = modRateCharges;
```

Comment:
This should only set state.modRate, to either modRate/modRteCharges, depending on what expirationTime was used.